src/elements/table/Table.js

```diff
@@ -331,16 +475,91 @@ export class NovoTable {
         this.emitSelected(selected);
     }
 
+	/**
+     * @name emitSelected
+     * @param selected
+     */
     emitSelected(selected) {
         this.onRowSelect.emit(selected.length, selected);
     }
 
+	/**
+     * @name rowClickHandler
+     * @param row
+     */
     rowClickHandler(row) {
         if (this.config.rowSelect) {
             this.activeId = row.id || 0;
             this.onRowClick.emit(row);
         }
     }
+
+    /**
+     * @name setDateOptions
+     * @param rowData
+     * @returns {Array}
+     */
+    setDateOptions(rowData) {
+        let range = [];
+        // Get max and min
+        rowData.forEach(row => {
+            range.push(row._dateDifference);
+        });
+        range.sort();
+        let minDate = range[0];
+        let maxDate = range[range.length - 1];
+        let dateOptions = [];
+        // For performance these could be staggered mathematically, but the order of the options needs to be preserved
+        if (maxDate > 91) {
+            dateOptions.push({ label: 'Beyond 90 Days', min: 90 });
+        }
+        if (maxDate >= 90) {
```

Comment:
@krsween do these dateOptions have to be hard coded here? could this possibly be the default options that are provided and just expect the structure of the dateOptions list to be a certain way (this would be provided by the user of novo-table)?